sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -77,14 +77,18 @@
 # pylint: disable=wrong-import-order, wrong-import-position
 try:
   from apitools.base.py.exceptions import HttpError
+  from google.cloud import bigquery as gcp_bigquery
 except ImportError:
+  gcp_bigquery = None
   HttpError = None
 # pylint: enable=wrong-import-order, wrong-import-position
 
 _LOGGER = logging.getLogger(__name__)
 
 
-@unittest.skipIf(HttpError is None, 'GCP dependencies are not installed')
+@unittest.skipIf(
+    HttpError is None or gcp_bigquery is None,
```

Comment:
Why do we need apitools and gcp_bigquery at the same time? (Are not they 2 mutually exclusive clients?)

Reply:
currently we've only migrated streaming inserts functionality to gcp_bigquery. All other functionality still works using apitools.